tests/helpers/loadCAR.ts

```diff
@@ -13,7 +13,7 @@ export async function loadCAR(filepath: string, ipfs: IPFS): Promise<{ roots: CI
     const blockIterator = await CarBlockIterator.fromIterable(inStream)
     for await (const block of blockIterator) {
       cids.push(block.cid)
-      await ipfs.block.put(block.bytes, { cid: block.cid })
+      await ipfs.block.put(block.bytes, { version: 1, mhtype: "sha2-256" })
```

Comment:
I think this code has issues, because it'll strip away the IPLD codec from all CIDs and turn all blocks to "raw" blocks implicitly (this code is probably from the wnfs2 branch).

We should totally get rid of this whole thing and just use the newly implemented [`ipfs.dag.import` api](https://github.com/ipfs/js-ipfs/blob/master/docs/core-api/DAG.md#ipfsdagimportsource-options).

Reply:
Aight! I'll see if I can use that `import` function 👀